stable/sonarqube/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: sonarqube
 description: Sonarqube is an open sourced code quality scanning tool
-version: 3.1.0
+version: 3.1.1
```

Comment:
Please update the minor version as this adds new features.

Reply:
Done